### PR TITLE
only for demonstration purposes, don't merge

### PR DESCRIPTION
**Pull request by tsvrko**

### CODE REVIEW

File: sources/zafira-web/angular/client/app/_testruns/testrun.controller.js [hunk 1]
```diff
@@ -1414,18 +1417,12 @@
...
-            var version = "v." + array[1];
-            if (array.length == 2) {
-                return version;
-            }
-            else {
-                return null;
+        var splitPlatform = function (string) {
+            var version = null;
+            if (string && string.split(' ').length > 1) {
+                version = "v." + string.split(' ')[1]
```
Comment (vdelendik) [inline sources/zafira-web/angular/client/app/_testruns/testrun.controller.js]: Why do we need “v.”?

Comment (tsvrko) [reply sources/zafira-web/angular/client/app/_testruns/testrun.controller.js]: just for display near platform icon

Comment (tsvrko) [reply sources/zafira-web/angular/client/app/_testruns/testrun.controller.js]: screen shot 2018-08-14 at 11 38 24 am

Comment (vdelendik) [reply sources/zafira-web/angular/client/app/_testruns/testrun.controller.js]: ah, that's js... all good

**Pull request closed without merging**